spec/vagrant-cloudstack/config_spec.rb

```diff
@@ -90,6 +91,14 @@
         instance.finalize!
         instance.send(attribute).should == "foo"
       end
+
+    end
+
+    it 'should not default pf_open_firewall if overridden' do
```

Comment:
Good point! Since boolean only have two possible values, the positive test is the dual that is already exercised in the default configs test.